docs/guides/getting-started/quick-start/README.md

```diff
@@ -0,0 +1 @@
+# Quick Start
```

Comment:
I think this whole file can get removed, just land people on HTML/CSS/JS by default

Reply:
well, if we plan to have more qs guides an index page (with a list of guides) would make sense, right? especially on mobile where it's a little bit annoying to navigate the menu

Reply:
Take a look at it now, I made an autogenerated category page. I tried it in another section before and it didn't work, but seems to be working now

Reply:
i like it 👍